Sofa/Component/IO/Mesh/tests/MeshExporter_test.cpp

```diff
@@ -49,9 +49,17 @@ using sofa::helper::system::FileSystem ;
 
 using ::testing::Types;
 
-#include <filesystem>
+#if __has_include(<filesystem>)
+  #include <filesystem>
+  namespace fs = std::filesystem;
+#elif __has_include(<experimental/filesystem>)
+  #include <experimental/filesystem> 
+  namespace fs = std::experimental::filesystem;
+#else
+  error "Missing the <filesystem> header."
+#endif
 namespace {
-std::string tempdir = std::filesystem::temp_directory_path().string() ;
+std::string tempdir = fs::temp_directory_path().string() ;
```

Comment:
```suggestion
const std::string tempdir = sofa::helper::system::FileRepository::getTempPath() ;
```

I suggest to use directly this utility function. That would avoid the boilerplate code for the includes. The same apply in the other test files.